src/main/java/org/sharesquare/hub/configuration/SecurityConfiguration.java

```diff
@@ -97,7 +104,10 @@ protected void configure(HttpSecurity http) throws Exception {
 				.mvcMatchers("/v3/api-docs/**", "/swagger-ui.html", "/swagger-ui/**").permitAll() // Swagger UI
 				.antMatchers("/h2-console/**").permitAll() // H2 Web Console
 				.antMatchers("/actuator/**").permitAll()
-				.mvcMatchers("/offers/**", "/targetSystems/**").hasAuthority(SCOPE_PREFIX + scope)
+				.mvcMatchers("/offers/**").hasAuthority(SCOPE_PREFIX + offersScope)
+				.mvcMatchers(GET, "/targetsystems/**").hasAuthority(SCOPE_PREFIX + offersScope)
```

Comment:
/targetsystems with offersScope ? Thats a typo I guess

Reply:
As I understood it, the client with the offers scope is allowed to get a list of the available target systems he can choose from. And the target system is only allowed to create or delete a target system entry.